sphinxcontrib/autodoc_pydantic/directives/autodocumenters.py

```diff
@@ -324,10 +334,43 @@ def add_collapsable_schema(self):
 
         schema_rest = self._convert_json_schema_to_rest(schema)
         source_name = self.get_sourcename()
-
         for line in schema_rest:
             self.add_line(line, source_name)
 
+    def add_erdantic_figure(self, collapsed: bool):
+        """Adds an erdantic entity relation diagram to the doc of an
+        pydantic model.
+
+        """
+        source_name = self.get_sourcename()
+        if erd is None:
+            error_msg = 'erdantic is not installed, you need to install ' \
+                'it before creating an Entity Relationship Diagram for ' \
+                'f{self.fullname}. See ' \
+                'https://autodoc-pydantic.readthedocs.io/' \
+                'en/stable/users/installation.html'
+            raise RuntimeError(error_msg)
+
+        # Graphviz [DOT language](https://graphviz.org/doc/info/lang.html)
+        figure_dot = erd.to_dot(self.object)
+        lines = [
+            ".. graphviz::",
+            "",
+        ]
+        lines += ['   ' + line
+                  for line in figure_dot.replace('\t', '   ').split('\n')]
+        lines.append("")
```

Comment:
For my taste, I would like to make this part a bit more concise:

```python
figure_dot = erd.to_dot(self.object).replace('\t', '   ').split('\n')
lines_dot = ['   ' + line for line in figure_dot]
lines = [".. graphviz::", ""] + lines_dot + [""]
```

Reply:
done in 25af89a